traitcuration/settings.py

```diff
@@ -89,6 +96,34 @@
     }
 }
 
+# Auth
+AUTH_USER_MODEL = 'traits.User'
+
+SOCIALACCOUNT_PROVIDERS = {
+    'google': {
+        'SCOPE': [
+            'profile',
+            'email',
+        ],
+        'AUTH_PARAMS': {
+            'access_type': 'online',
+        }
+    }
+}
+
+AUTHENTICATION_BACKENDS = [
+    # Needed to login by username in Django admin, regardless of `allauth`
+    'django.contrib.auth.backends.ModelBackend',
+    # `allauth` specific authentication methods, such as login by e-mail
+    'allauth.account.auth_backends.AuthenticationBackend',
+]
+
+ACCOUNT_EMAIL_REQUIRED = True
```

Comment:
Perhaps it's worth mentioning in a comment that those configuration options are of the “allauth” package

Reply:
Agreed, I will add that.